convert2rhel/main.py

```diff
@@ -60,6 +60,9 @@ def main():
         loggerinst.task("Prepare: End user license agreement")
         user_to_accept_eula()
 
+        loggerinst.task("Prepare: Clear yum version locks")
+        pkghandler.clear_yum_versionlock()
```

Comment:
Please move this below the `subscription.rhn_reg_file.backup()`. Doing it as a first thing, before even checking if the system is possible to convert (gathering system info), is not right.